var/spack/repos/builtin/packages/mesa18/package.py

```diff
@@ -44,9 +44,7 @@ class Mesa18(AutotoolsPackage):
     variant("llvm", default=True, description="Enable LLVM.")
     variant(
         "swr",
-        values=spack.variant.DisjointSetsOfValues(
-            ("none",), ("auto",), ("avx", "avx2", "knl", "skx")
-        )
+        values=DisjointSetsOfValues(("none",), ("auto",), ("avx", "avx2", "knl", "skx"))
```

Comment:
This change looks quite unrelated to me?

Reply:
It is related. We didn't import `spack.variant` (see https://github.com/spack/spack/pull/40262#discussion_r1380648533), but we are forwarding the name `DisjointSetOfValues` to `spack.package` which is imported at the top. `spack.package` is a collection of directives and other methods, classes etc. that are frequently used in package recipes.

So I fixed the missing import, by referring to the same name imported from elsewhere.

Reply:
Okay, got it. The same as above: Maybe put this in a preparatory PR that gets merged before this one?